core/src/tests/test-parachain/src/lib.rs

```diff
@@ -185,25 +189,26 @@ parameter_types! {
 
 impl cumulus_pallet_parachain_system::Config for Runtime {
 	type CheckAssociatedRelayNumber = cumulus_pallet_parachain_system::RelayNumberStrictlyIncreases;
-	type DmpMessageHandler = DmpQueue;
+	type DmpQueue = EnqueueWithOrigin<(), sp_core::ConstU8<0>>;
 	type OnSystemEvent = ();
 	type OutboundXcmpMessageSource = XcmpQueue;
 	type ReservedDmpWeight = ReservedDmpWeight;
 	type ReservedXcmpWeight = ReservedXcmpWeight;
 	type RuntimeEvent = RuntimeEvent;
-	type SelfParaId = parachain_info::Pallet<Runtime>;
+	type SelfParaId = staging_parachain_info::Pallet<Runtime>;
+	type WeightInfo = ();
 	type XcmpMessageHandler = XcmpQueue;
 }
 
 // DMP
 
 impl cumulus_pallet_dmp_queue::Config for Runtime {
-	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+	type DmpSink = EnqueueWithOrigin<(), sp_core::ConstU8<0>>;
```

Comment:
We can remove `dmp_queue` from here because it has been substituted by `MessageQueue`